EventListener/Login.php

```diff
@@ -63,6 +67,21 @@ public function setCustomerId($value)
         $this->options['customerId'] = $value;
     }
 
+    protected function getUserIdentifier(TokenInterface $token)
+    {
+        $tokenUser = $token->getUser();
+
+        if ($tokenUser instanceof eZUser || !$tokenUser instanceof UserInterface) {
+            return $tokenUser->getAPIUser()->id;
+        }
+
+        if (method_exists($tokenUser, 'getId')) {
+            return $tokenUser->getId();
+        }
```

Comment:
So just to clarify this, which interface is this? (`getId `)
If none it should probably not be there, unless we add one.